content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,315 @@
+----------
+
+When offering potential employees a position, there can be certain advantages set in Odoo to make
+the offer more appealing other than the salary, such as extra time off, access to a company car,
+reimbursement for a phone or internet, etc. Any specific advantage is listed in the
+:guilabel:`Advantages` section of the configuration menu. To see the advantages, go to
+:menuselection:`Payroll --> Configuration --> Advantages`.
+
+.. image:: configuration/advantages.png
+   :align: center
+   :alt: Settings available for payroll.
+
+To make a new advantage, click the :guilabel:`Create` button. Enter the information in the fields,
+then click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
+the entry.
+
+.. image:: configuration/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
+
+Personal info
+-------------
+
+Every employee in Odoo has an `employee card` that includes all of their personal information,
+resume, work information, and documents.
+
+.. note::
+   An employee card can be thought of as an employee file.
+
+The *Personal Information* section lists all of the fields that are available to enter on the
+employee's card. To access this section, go to :menuselection:`Payroll --> Configuration -->
+Personal Information`.
+
+.. image:: configuration/personal-info.png
+   :align: center
+   :alt: Personal information that appear on employee cards to enter.
+
+To edit an entry, click on the line. Then, click the :guilabel:`Edit` button, and modify the entry.
+When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel the
+edits.
+
+To create a new entry, click :guilabel:`Create`. A box appears with all the fields to enter.
+
+.. image:: configuration/personal-new.png
+   :align: center
+   :alt: New personal information entry.
+
+The two most important fields are :guilabel:`Is Required` and :guilabel:`Display Type`. Checking
+the :guilabel:`Is Required` box will make the field mandatory on the employee's card.
+
+The :guilabel:`Display Type` drop-down menu allows for the information to be entered in a variety of
+ways, from a text box, a customizable radio button, a check box, a document, and more.
+
+Once the information is entered, click the :guilabel:`Save` button to save the entry.
+
+Resume
+------
+
+Click on :menuselection:`Payroll --> Configuration --> Resume` to view the list of items that are
+pulled from online job applications housed in the resume section. These fields are populated when a
+localization is selected. The default items are gross, monthly equivalent, employer cost, and net.
+
+.. image:: configuration/resume.png
+   :align: center
+   :alt: The list of items in the resume section related to payroll.
+
+There should be no need to modify these as it will affect payroll an accounting, but edits can be
+made. Click on an individual entry to see the details for the entry. Click :guilabel:`Edit` to make
```

Comment:
```suggestion
There should be no need to modify these as it will affect payroll and accounting, but edits can be
made if needed. Click on an individual entry to see the details for the entry. Click :guilabel:`Edit` to make
```